### PR TITLE
Adjust maintainers list

### PR DESCRIPTION
**Pull request by dzhulgakov**
Adds new people and reorders sections to make more sense

### CODE REVIEW

**Comment by soumith** [comment]
@xuhdev let us know if you're comforable being listed for build system
@vishwakftw let us know if you are comfortable with your name also listed under linear algebra

**Comment by xuhdev** [comment]
@soumith Yes, please. Thanks!

File: docs/source/community/persons_of_interest.rst [hunk 1]
```diff
@@ -15,116 +15,136 @@ General Maintainers
...
+
+-  Vincent Quenneville-Belair (`vincentqb <https://github.com/vincentqb>`__)
+-  Soumith Chintala (`soumith <https://github.com/soumith>`__)
+
+Autograd Engine
+~~~~~~~~~~~~~~~
+
+-  Edward Yang (`ezyang <https://github.com/ezyang>`__)
+-  Alban Desmaison (`alband <https://github.com/alband>`__)
+-  (sunsetting) Adam Paszke (`apaszke <https://github.com/apaszke>`__)
```
Comment (apaszke) [inline docs/source/community/persons_of_interest.rst]: FWIW I'm reading every single autograd PR, even if I don't comment on it, so I wouldn't agree with that 😬

File: docs/source/community/persons_of_interest.rst [hunk 2]
```diff
@@ -15,116 +15,136 @@ General Maintainers
...
+XLA
 ~~~
 
--  Will Feng (`yf225 <https://github.com/yf225>`__)
--  (sunsetting) Peter Goldsborough
-   (`goldsborough <https://github.com/goldsborough>`__)
+-  Ailing Zhang (`ailzhang <https://github.com/ailzhang>`__)
+-  Gregory Chanan (`gchanan <https://github.com/gchanan>`__)
+-  Davide Libenzi (`dlibenzi <https://github.com/dlibenzi>`__)
+-  Alex Suhan (`asuhan <https://github.com/asuhan>`__)
```
Comment (apaszke) [inline docs/source/community/persons_of_interest.rst]: IIUC Alex is no longer working on the XLA bridge

**Comment by vishwakftw** [comment]
> @vishwakftw let us know if you are comfortable with your name also listed under linear algebra
Yes, very much. Thank you!

File: docs/source/community/persons_of_interest.rst (hunk 1 above)
Comment (dzhulgakov) [reply docs/source/community/persons_of_interest.rst]: Ok, sure, it was @soumith's suggestion :-P

**Review by soumith — APPROVED** [review]
go go go!

File: docs/source/community/persons_of_interest.rst (hunk 1 above)
Comment (apaszke) [reply docs/source/community/persons_of_interest.rst]: Thanks @soumith 😊

**Pull request closed without merging**